src/main/java/io/cryostat/net/SslConfiguration.java

```diff
@@ -64,9 +64,13 @@ public class SslConfiguration {
         this.logger = logger;
 
         if (env.hasEnv(Variables.DISABLE_SSL)) {
-            strategy = new NoSslStrategy();
-            logger.info("Selected NoSSL strategy");
-            return;
+            String disableSslValue = env.getEnv(Variables.DISABLE_SSL);
+            boolean disableSsl = Boolean.parseBoolean(disableSslValue); // Parse the value as a boolean
```

Comment:
Part of this bug is that `env.hasEnv(Variables.DISABLE_SSL)` behaves differently than what [`entrypoint.bash`](https://github.com/cryostatio/cryostat/blob/c675a2e113a3d8fc25136ec52919c7a126e31290/src/main/extras/app/entrypoint.bash#L144) does, given the same inputs (environment variable values).

How does `Boolean.parseBoolean()` compare to the `bash` `var = "true"` check? They should behave identically given the same inputs, or else we will still have buggy behaviour due to the mismatch.

Reply:
I think there is still a discrepancy between `var = "true"` and `var.equalsIgnoreCase("true")`.

Reply:
are you considering in cases where there are white spaces after "true"? example ``` var = "true  "```?

Reply:
No, I'm considering `CRYOSTAT_DISABLE_SSL="TRUE"`, for example.

Reply:
1. so it should work for all cases.. with ```CRYOSTAT_DISABLE_SSL="TRUE"```
2. ```CRYOSTAT_DISABLE_SSL="TRUE   "```
3. ```CRYOSTAT_DISABLE_SSL="tRue" ```
4. ```CRYOSTAT_DISABLE_SSL="true  "```?

Reply:
if needed ignore/remove the whitespace I know i can use trim()

Reply:
What I am saying is that the Java code and the `entrypoint.bash` need to treat all of these cases the same. Whether that means they do or do not accept (ie. disable SSL) each of those cases is another matter, but it needs to be consistently handled.